SPECS/golang/golang-1.15.spec

```diff
@@ -31,18 +31,18 @@ Go is an open source programming language that makes it easy to build simple, re
 %prep
 # Setup go 1.4 bootstrap source
 tar xf %{SOURCE1} --no-same-owner
-patch -Np1 --ignore-whitespace < /usr/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
+patch -Np1 --ignore-whitespace < %{_prefix}/src/mariner/SOURCES/go14_bootstrap_aarch64.patch
 mv -v go go-bootstrap
 
 %setup -q -n go
 
 %build
 # Build go 1.4 bootstrap
-pushd /usr/src/mariner/BUILD/go-bootstrap/src
+pushd %{_prefix}/src/mariner/BUILD/go-bootstrap/src
```

Comment:
Please don't use these macros (regardless of what the linter says) for internal package directory paths. They are meant for the final system the package gets installed on and are independent of the folder structure from in a single source tarball.